tests/emscripten_log/emscripten_log.cpp

```diff
@@ -154,19 +149,19 @@ void DoubleTest() {
   PrintDoubleStuff(12.3456789, 9.12345678);
 }
 
-int main()
-{
-	int test = 123;
-	emscripten_log(EM_LOG_FUNC_PARAMS | EM_LOG_CONSOLE, "test print %d\n", test);
+int main() {
+  int test = 123;
+  emscripten_log(EM_LOG_FUNC_PARAMS | EM_LOG_CONSOLE, "test print %d\n", test);
 
-	Foo<int>();
+  Foo<int>();
 
   DoubleTest();
 
-#ifdef REPORT_RESULT
```

Comment:
Where was this defined? Is it still defined somewhere? Can we get rid of it?

Reply:
Sorry I should have given more context.

Traditionally when you run browser test `self.btest` the test running will inject the `report_result.h` header (using `-include`) and expect you to call `REPORT_RESULT`.   However, in recent months I've added `self.btest_exit` which just relies on the exit code instead.    It doesn't inject any headers and it doesn't use this macro.  See #12928 and #12977.